{
  "singleton_construct_error": "Impossibile creare un'istanza della classe singleton %(classname)s",
  "create_local_item_error": "Errore in createLocalItem: ",
  "error_filtering_remote_items": "Errore durante il filtraggio dell'elenco degli elementi remoti in listModifiedRemoteItemNames",
  "push_error_creating_item": "Errore push durante la creazione dell'elemento: ",
  "push_error_updating_item": "Errore push durante l'aggiornamento dell'elemento: ",
  "pushed_item_error": "Errore dell'elemento sottoposto a push: ",
  "pushed_item_retry": "Ripetere il push dell'elemento '%(name)s'. Errore: %(message)s",
  "invalid_path": "Percorso non valido: %(path)s",
  "timed_out_closing_read_stream": "Si è verificato un timeout durante l'attesa della chiusura del flusso di lettura",
  "remote_asset_not_found": "Asset remoto non trovato: ",
  "cannot_push_item": "Impossibile eseguire il push dell'elemento '%(name)s'.",
  "cannot_pull_item": "Impossibile eseguire il pull dell'elemento '%(name)s'.",

  "deleted_original_file": "File originale eliminato: %(old_name)s nuovo file: %(new_name)s",
  "error_cancelling_job": "Errore durante l'annullamento del lavoro: id %(id)s",
  "error_fs_get_item": "Errore nel getItem del file system",
  "error_fs_get_filestats": "Errore in getFileStats file system per %(name)s.",
  "error_parsing_item": "Errore durante l'analisi del file: %(name)s. %(message)s",
  "get_items_error": "Errore durante il richiamo degli elementi %(service_name)s: ",
  "save_item_write_failed": "saveItem non è riuscito a scrivere sul file %(path)s",
  "save_item_write_failed_bad_path": "saveItem non è riuscito a scrivere sul file %(path)s, percorso errato",
  "close_stream_failed": "closeStream non riuscito con errore: ",
  "error_options_file": "Errore durante la lettura del file delle opzioni di configurazione: %(filename)s. %(error)s",

  "name_required": "È necessario specificare il nome.",
  "category_exists": "Una categoria con nome %(name)s esiste già.",
  "template_required": "È necessario specificare il modello.",
  "cannot_get_asset": "Impossibile richiamare l'asset: %(path)s; codice stato: %(code)s",
  "deleted_asset": "Eliminato l'asset con ID: %(id)s",
  "create_asset_metadata_error": "Errore durante la creazione dei metadati dell'asset: ",
  "deleted_item": "Eliminato l'elemento con ID: %(id)s",
  "deleted_item_retry": "Ripetere l'eliminazione dell'elemento '%(name)s'. Errore: %(message)s",
  "get_item_error": "Errore durante il richiamo dell'elemento %(service_name)s: ",
  "create_item_error": "Errore durante la creazione dell'elemento %(service_name)s: ",
  "update_item_error": "Errore durante l'aggiornamento dell'elemento %(service_name)s: ",
  "delete_item_error": "Errore durante l'eliminazione dell'elemento %(service_name)s: ",
  "delete_rendition_error": "L'eliminazione dei rendering non è supportata. ID: %(id)s, opzioni: %(opts)s",
  "error_tenant_map": "Errore in loadTenantMap: ",
  "error_save_hashes": "Errore in saveHashes: ",
  "error_update_hashes": "Errore in updateHashes: ",
  "error_remove_hashes": "Errore in removeHashes: ",
  "error_remove_all_hashes": "Errore in removeAllHashes: ",
  "error_set_last_pull_timestamp": "Errore in setLastPullTimestamp: ",
  "error_set_last_push_timestamp": "Errore in setLastPushTimestamp: ",
  "error_get_hashes_for_file": "Errore in getHashesForFile: ",

  "error_unsupported_endpoint": "L'API %(service_name)s non supporta l'endpoint %(endpoint)s.",
  "unknown_error": "Errore sconosciuto",
  "service_unavailable": "Un servizio IBM Watson Content Hub non è attualmente disponibile.  ",
  "service_error": "Un servizio IBM Watson Content Hub sta notificando difficoltà tecniche.  ",
  "service_response_error": "Un servizio IBM Watson Content Hub ha restituito il codice di errore %(code)d e il messaggio di stato: %(message)s",
  "operation_failed": "Operazione non riuscita con errore: ",
  "async_parallel_limit_error": "Chiamata parallelLimit asincrona non riuscita con errore: ",
  "async_parallel_limit_results": "Risultati di parallelLimit asincrono: ",
  "please_try": "Riprovare in seguito.  Se il problema persiste, contattare il supporto clienti e includere le informazioni del log disponibili in %(log_dir)s",
  "conflict":"Il servizio ha riscontrato e restituito una condizione di errore durante l'elaborazione della richiesta per questa risorsa.",
  "login_missing_authn":"Il servizio di login non ha risposto con informazioni di autenticazione valide.",
  "digest_mismatch": "digest %(cli_digest)s per richiamato %(asset)s non corrispondente al digest server %(server_digest)s.",
  "retry_failed_request": "Una richiesta (%(id)d) non riuscita verrà ritentata. %(message)s",
  "retry_failed_retry": "Una richiesta (%(id)d) ritentata (attempt %(attempt)d) non è riuscita entro il numero massimo di tentativi. %(message)s",
  "retry_next_attempt": "Una richiesta (%(id)d) verrà ritentata (attempt %(attempt)d) in %(delay)d millisecondi.",
  "retry_success": "Una richiesta (%(id)d) ritentata è riuscita dopo %(attempt)d tentativi.",
  "retry_max_attempts": "Una richiesta (%(id)d) è stata ritentata il numero massimo di volte."
}

